sycl/test/spec_const/spec_const_hw.cpp

```diff
@@ -5,7 +5,7 @@
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 // TODO: re-enable after CI drivers are updated to newer which support spec
 // constants:
-// XFAIL: linux && opencl
+// XFAIL: linux && opencl && !gpu && !cpu
```

Comment:
Another idea:
```suggestion
// XFAIL: linux && opencl && accelerator
```

Reply:
This will make the test fail as expected though we do not get test coverage if it actually passes for CPU and GPU.

Reply:
I wonder if to duplicate the test to make an accelerator-specific version with an XFAIL in it?